posthog/session_recordings/test/test_session_recording_playlist.py

```diff
@@ -632,3 +658,67 @@ def test_add_remove_static_playlist_items(self):
             ).count()
             == 0
         )
+
+    def test_filters_playlist_by_type(self):
```

Comment:
if you add `@snapshot_postgres_queries` here then we can check if I'm right about the multiple SQL queries above

```suggestion
    @snapshot_postgres_queries
    def test_filters_playlist_by_type(self):
```

Reply:
ping here @veryayskiy

Reply:
👋 I really do want to add the postgres snapshot to this test so we can see what queries we're generating :)